src/Kushy/SymbolLoader.cs

```diff
@@ -77,6 +107,10 @@ public async Task<DatabaseSymbol> LoadDatabaseAsync(string databaseName, string
             var materializedViews = await LoadMaterializedViewsAsync(connection, databaseName, throwOnError, cancellationToken).ConfigureAwait(false);
             var functions = await LoadFunctionsAsync(connection, databaseName, throwOnError, cancellationToken).ConfigureAwait(false);
 
+            if(tables.Count==0 && externalTables.Count==0 && materializedViews.Count==0 && functions.Count==0)
```

Comment:
Are you just assuming the database is bad because it has no schema defined?

Reply:
these changes are done one version prior to the latest version hence removed.